model_analyzer/config/run/model_run_config.py

```diff
@@ -117,6 +128,26 @@ def is_legal_combination(self):
             )
         return legal
 
+    def is_ensemble_model(self) -> bool:
+        """
+        Returns true if the model_config is an ensemble model
+        """
+        return self._model_config.is_ensemble()
+
+    def get_ensemble_subconfig_names(self) -> Optional[List[str]]:
+        """
+        Returns list of Ensemble Subconfig names
+        """
+        return self._model_config.get_ensemble_submodels(
+        ) if self._model_config.is_ensemble() else None
```

Comment:
is it safer to return empty list instead of None?

Reply:
No, None is fine and is equivalent to an empty list when checking in an if statement

Reply:
if (Y):   -- works for both none and empty list
for X in Y:  -- works for empty list, but not for None.

Is there a case where None is better? I hate having None all over our code and needing to make a bunch of types Optional.

Reply:
This doesn't change the need for optional. Both None and [] will require it.

Reply:
But, I've changed this to be []